app/enterprise/2.1.x/developer-portal/administration/application-registration/azure-oidc-config.md

```diff
@@ -0,0 +1,141 @@
+---
+title: Set up External Portal Application Authentication with Azure AD and OIDC
+---
+
+## Overview
+
+These instructions help you set up Azure AD as your third-party identity provider
+for use with the Kong OIDC and Portal Application Registration plugins.
+
+### Prerequisites
+
+- The `portal_app_auth` configuration option is configured for your OAuth provider
+  and strategy (`kong-oauth2` or `external-oauth2`). See
+  [Configure the Authorization Provider Strategy](/enterprise/{{page.kong_version}}/developer-portal/administration/application-registration/#portal-app-auth) for the Portal Application Registration plugin.
+
+## Create an Application in Azure
+
+1. Within Azure, go to the **App Registrations** service and register a new application.
+
+   ![Azure App Registrations](/assets/images/docs/dev-portal/ms-azure-app-reg.png)
+
+1. In **Certificates & secrets**, create a Client secret and save it in a
+   secure location. You can only view the secret once.
+
+1. Under **Manifest**, update `accessTokenAcceptedVersion=2` (default is null).
+   The JSON for your application should look similar to this example:
+
+  ![Azure Manifest](/assets/images/docs/dev-portal/azure-manifest.png)
+
+## Create a Service and a Route in Kong
+
+1. Create a Service example:
+
+   ```bash
```

Comment:
Most of the examples in our docs are in cURL, or in both cURL and HTTPie. We don't normally do HTTPie only. Can you turn these examples into cURL and HTTPie tabs, at the section level (right under the header)?

For example, this command would be the following in cURL:
```
curl -i -X PUT http://<admin-server>:8001/services/httpbin-service-azure2 \
   --data 'url=https://httpbin.org/'
```
You don't absolutely need `-i`, but it makes the output easier to read.

Reply:
@lena-larionova yes we can show cURL as well. I had mentioned that to @aaronhmiller. The OIDC example uses Kong Manager GUI and doesn't show any commands. So we need to be consistent (something for me to keep in mind for the 3rd party/OIDC updates PR). I'm not sure if we'll get to publish/keep these IdP-specific examples.

I don't have anything set up in azure. I'll need someone to look over my guesses at converting httpie -> curl.

Reply:
I think your example above has an extraneous 2 in `azure2`